modules/reactor/src/meshgenerators/PatternedCartesianMeshGenerator.C

```diff
@@ -792,15 +812,19 @@ PatternedCartesianMeshGenerator::generate()
     }
   }
 
-  // Check if OUTER_SIDESET_ID is really the external boundary. Correct if needed.
+  // Check if stitching_boundary_id is really the external boundary. Correct if needed.
```

Comment:
Previous, we are sure that `OUTER_SIDESET_ID` should be the external boundary. So this step is correction. However, for a user specified boundary id, I am not sure if we want to modify it. We should describe this behavior in doc.

Reply:
added to the doco